### ISSUE TITLE
covariance structure?

### ISSUE THREAD

**Issue by ckm14** [mcfanda/gamlj#23]
First of all, thanks for the great module.

I want to make sure I'm replicating my analyses on SPSS properly (for all my SPSS using collaborators...unfortunately).

In SPSS MIXED, you can specify the repeated covariance structure and the random effect covariance structure. I see that there are two options either VC or UN in this module. However, I am not replicating my results across both platforms. I wonder if it has to do with the repeated covariance that SPSS offers? I don't see that this module specifies this, or is it within lmer somewhere that I'm not seeing? Any insight would be helpful...

**Comment by mcfanda** [comment]
Hi,
please check out version 0.9.2 where some fixes have been pushed for the mixed model. You should be able to replicate (some) SPSS results (up to reasonable rounding) for UN and VC. The latter is obtained in GAMLj by unselecting "correlated effects". Please notice, however, that several considerations apply  here:
Jamovi GAMLj implements R lme4 lmer estimation, with "lmerTest" and "car" packages for several additional tests and utilities. To replicate SPSS results in GAMLj, they should be replicable in  R, which is not always the case.
In cases where SPSS and R give the same results,  different results in GAMLj and SPSS are  mostly due to different defaults in the two softwares. 

When you have random effects of continuous variables, the only reason you may get different results in GAMLj and SPSS is because GAMLj by default centers the continuous variables (which usually helps convergence) whereas SPSS does not. For the sake of comparison, in GAMLj you can choose the scaling of the covariates in "covariate scaling"  and set it to "none". 

If you have random effects of categorical variables (in SPSS the ones put in "Factors"), SPSS and GAMLj give different results for random variances because SPSS and R give different results in the first place.  In these cases, you can check that both softwares give the same residual variance. If they do, the estimation of the fixed effects will be very similar in both softwares, and so will be the inferential statistics. The random variances, however, will be different. 

Bottom line, please update to 0.9.2 and check out the results in GAMLj and SPSS. Please let me know your thoughts
marcello
